apps/style/netsim/style.scss

```diff
@@ -24,7 +24,6 @@ $wide-column-width: 121px;
   user-select: $val;
 }
 
-// Reduce top margin to help fit everything above the fold
 .full_container {
   padding-top: 2px;
```

Comment:
You inspired me to remove this and a bunch of other rules that I _think_ got pulled over from calc back when I started NetSim. :+1: